tests/test-dirs/locate/locate-stdlib.t

```diff
@@ -0,0 +1,45 @@
+  $ cat >dune-project <<EOF
+  > (lang dune 2.0)
+  > EOF
+
+  $ cat >s.ml <<EOF
+  > module Foo = Set.Make(struct
+  >   type t
+  >   let compare _ _ = 0
+  > end)
+  > type t = Foo.t
+  > EOF
+
+  $ cat >dune <<EOF
+  > (executable (name s))
+  > EOF
+
+  $ dune build ./s.exe
+
+Should jump to set.ml:
+  $ $MERLIN single locate -look-for ml -position 5:13 -filename ./s.ml < ./s.ml
```

Comment:
That's the kind of test whose output is likely going to be unstable: `set.ml` might change with each new release of ocaml, moving a bit the position returned by locate, creating noise in the diff we'll have to check.

What are you trying to test here and can we test it without adding "external dependencies"?